src/csharp/Microsoft.Spark/Sql/DataFrame.cs

```diff
@@ -70,6 +70,15 @@ internal DataFrame(JvmObjectReference jvmObject)
             Console.WriteLine(
                 (string)((JvmObjectReference)_jvmObject.Invoke("schema")).Invoke("treeString"));
 
+        /// <summary>
+        /// Prints the schema up to the given level to the console in a nice tree format.
+        /// </summary>
+        [Since(Versions.V3_0_0)]
+        public void PrintSchema(int level) =>
+            Console.WriteLine(
+                (string)((JvmObjectReference)_jvmObject.Invoke("schema"))
+                .Invoke("treeString", level));
```

Comment:
indentation looks off?

Reply:
Changed, please let me know if that looks better!

Reply:
You can just do the indentation looks funny.

```csharp
        public void PrintSchema(int level)
        {
            var schema = (JvmObjectReference)_jvmObject.Invoke("schema");
            Console.WriteLine((string)schema.Invoke("treeString", level));
        }
```